src/data_storage/graph/edge_storage.ts

```diff
@@ -30,6 +32,16 @@ export default class EdgeStorage extends PostgresStorage{
         return EdgeStorage.instance
     }
 
+    public async CreateOrUpdateByActiveGraph(containerID: string, input: any | EdgesT, preQueries?: QueryConfig[], postQueries?: QueryConfig[]): Promise<Result<EdgesT>> {
```

Comment:
I believe the API calls using this function did not include a GraphID, while other functions local to the code do. Rather than reworking all the existing functionality around create/update for nodes and edges, I went the route of creating another path into creation/update that provides the current active graph ID. I think we discussed this previously, but let me know if you think a rework to merge this function with the CreateOrUpdate function would make sense.